### ISSUE TITLE
The generated picture is very blurry.

### ISSUE THREAD

**Issue by cjgogo27** [ByteDance-Seed/Bagel#259]
<img width="1921" height="1167" alt="Image" src="https://github.com/user-attachments/assets/1f8b612c-cdf6-4810-a628-49c2262dbb14" />
<img width="2453" height="939" alt="Image" src="https://github.com/user-attachments/assets/42434f9c-f2db-4ada-afcf-30b6ee45aa70" />
<img width="1006" height="638" alt="Image" src="https://github.com/user-attachments/assets/7dabce14-2480-48c5-8b7e-50dc9995fe47" />

**Comment by Plantian** [comment]
Hi, are you doing this task after finetuing? I also face with this problems

**Comment by cjgogo27** [comment]
I reconfigured the environment once, and it worked normally. However, I still don't know why it went wrong before.

**Comment by Plantian** [comment]
您是微調之後測試的嗎 我是做了微調訓練



发自我的iPhone


------------------ Original ------------------
From: Yang Chenjie ***@***.***&gt;
Date: Fri,Oct 10,2025 5:36 PM
To: ByteDance-Seed/Bagel ***@***.***&gt;
Cc: Plantian ***@***.***&gt;, Comment ***@***.***&gt;
Subject: Re: [ByteDance-Seed/Bagel] The generated picture is very blurry.(Issue #259)



cjgogo27 left a comment (ByteDance-Seed/Bagel#259)
 
I reconfigured the environment once, and it worked normally. However, I still don't know why it went wrong before.

**Comment by cjgogo27** [comment]
我没有微调

**Comment by BadaLeexxx** [comment]
> Hi, are you doing this task after finetuing? I also face with this problems

Hi Do you solve it？I meet the same problem after pretrain with bagel example data. @Plantian

**Comment by cjgogo27** [comment]
Actually, I didn't perform any fine-tuning when I encountered this issue. It was only afterward that I planned to do the fine-tuning. I reconfigured the environment once, and then it worked normally. It seems that the problem might have been related to the Flash package.

**Comment by yankai233** [comment]
You can try following the author's instructions exactly. I previously tried to be clever and installed the CUDA version of Torch myself, because I thought `pip install -r requirements.txt` would download the CPU version of Torch, and I encountered the same problem as you. I tried installing it directly with `pip install -r requirements.txt` (without manually installing the CUDA version of Torch), and to my surprise, it worked! The `requirements.txt` file even installed the version that could use CUDA, which amazed me.

**Comment by Plantian** [comment]
> > Hi, are you doing this task after finetuing? I also face with this problems
> 
> Hi Do you solve it？I meet the same problem after pretrain with bagel example data. [@Plantian](https://github.com/Plantian)

yes you can connect me with my emails.
